### DISCUSSION TITLE
recommendations for multi-cluster management

### DISCUSSION THREAD

**Discussion by pdeva** [argoproj/argo-rollouts#3401] [Q&A]
We have k8s clusters in multiple regions. is it possible to manage Rollouts for all of them through a single argo-rollouts dashboard? I realize that each cluster will still need to deploy its own argo-rollouts-controller pods to get the CRDs, but can they all report to a central dashboard. that way multiple regions could be managed from a single pane.

**Comment by kostis-codefresh** [comment]
I don't think that the dashboard currently supports what you ask.

**Comment by synchrone** [comment]
it seems to be achievable via an ArgoCD plugin, see https://ivanlee.me/cross-cluster-argo-rollouts/
